contracts/allowlist/README.md

```diff
@@ -0,0 +1,38 @@
+# Allowlist Smart Contract
+
+Accounts on the allowlist can use the API of this contract to 
+* activate a validator
+* deactivate a validator
+* vote to add an account to the allowlist
+* vote to remove an account from the allowlist
+* remove votes they have cast to add or remove an account
+* execute the vote count for an account to be added or removed
+
+For a vote to be successful more than 50% of the current members of the allowlist have to vote. 
+
+To get the votes counted and a new account being added or an account to be removed the function _countVotes needs to 
+be called.
```

Comment:
do we need to mention that only one vote/election can be "open" at any one time?

Reply:
That is not the case. You can vote at any time to add someone who is not on the allowlist, or remove someone who is. Votes are removed once the votes have been counted, or when the function removeVote function is called.